lib/peep.rb

```diff
@@ -0,0 +1,37 @@
+require 'pg'
+require 'date'
+
+class Peep
+  attr_reader :id, :title, :peep, :time
+
+  def initialize(id:, title:, peep:, time:)
+    @id = id
+    @title = title
+    @peep = peep
+    @time = time
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] = 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+    result = connection.exec("SELECT * FROM peep_manager")
+    result.map do |peep|
+      Peep.new(id: peep['id'], title: peep['title'], peep: peep['peep'], time:['time'])
```

Comment:
You may want to use the 'splat' operator here. Here is an example use:
```rb
def go(x:, y:)
end

point = { x: 100, y: 200 }
go(**point)
```

Reply:
Also... this may actually solve your "time" bug :) You have a typo at the end of this line.